### ISSUE TITLE
Staging Review finding: Unexpected focus behavior when uploading files

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93005]
### Acceptance criteria
- Verify this has been fixed as part of FMP claim update.

_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-10d CHAMPVA Application Form
**Feature:** new product

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When uploading files (eg. [/apply-form-10-10d/applicant-other-insurance-upload/](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-other-insurance-upload/0)), after uploading the first file, focus goes to the "Here's how you know" link in the "official website" banner.

This is an unexpected behavior for keyboard-only users, who now need to navigate through the whole page back to the document type select. Similarly, screen reader users are placed in an unexpected context and must figure out where they are, what happened, and how to get back.

When uploading subsequent files, focus is sent to the "Delete file" button. That's closer to the neighborhood where you would expect to be, but it requires backtracking to get to the required document type field. That's an inconvenience for keyboard-only users, and screen reader users may not realize that field exists until they fail to make their required selection.

<details><summary>Link, screenshot or steps to recreate</summary>

[video]

</details>

## Recommended action

I would expect focus to go somewhere that makes sense and is actionable. In this case, I think that's either the filename of the uploaded file in the card (using `tabindex="-1"` to send focus there), or to the document type `select` element.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by michaelclement** [comment]
@baltes This is visible in staging and ready for your review.

Per the recommendation, focus is returned to the document type `<select>` input; it does have a small delay before focus is returned due to the file upload needing to complete before focus can jump there.

https://github.com/user-attachments/assets/dfc9ecd7-c6eb-441a-bad3-6a7df732a8d2
